diff --git a/gidocgen/gidocmain.py b/gidocgen/gidocmain.py
index a3c97f2..ba516f0 100644
--- a/gidocgen/gidocmain.py
+++ b/gidocgen/gidocmain.py
@@ -98,7 +98,7 @@ class GIDocGenApp:
 
         # Add shared commands
         p.add_argument("-q", "--quiet", action="store_true", help="suppress messages except warnings")
-        p.add_argument("--fatal-warnings", action="store_true", help="whether warnings are fatal")
+        p.add_argument("--fatal-warnings", action="store_false", help="whether warnings are fatal")
 
         if add_args_func:
             add_args_func(p)
